test/powershell/Modules/Microsoft.PowerShell.Utility/xml.tests.ps1

```diff
@@ -2,15 +2,15 @@
     Context "Select-XML" {
         BeforeAll {
             $fileName = New-Item -Path 'TestDrive:\testSelectXml.xml'
-	        Push-Location "$fileName\.."
-	        "<Root>" | out-file -LiteralPath $fileName
-	        "   <Node Attribute='blah' />" | out-file -LiteralPath $fileName -Append
-	        "</Root>" | out-file -LiteralPath $fileName -Append
+            Push-Location "$fileName\.."
+            "<Root>" | out-file -LiteralPath $fileName
+            "   <Node Attribute='blah' />" | out-file -LiteralPath $fileName -Append
+            "</Root>" | out-file -LiteralPath $fileName -Append
```

Comment:
Tabs?

Reply:
It used to be tabs, and now is changed to spaces as in rest of the file.